aomi/seed.py

```diff
@@ -483,14 +526,35 @@ def approle(client, approle_obj, opt):
     client.create_role(name, **role_obj)
 
 
+def generated_key(key, opt):
```

Comment:
We aren't going to be including bits of `aomi` in third party applications to generate secrets... Which is what this comment seems to imply?